### ISSUE TITLE
Http validation failed on in_progress status

### ISSUE THREAD

**Issue by sergeysova** [chrnorm/deployment-status#8]
```yaml
      - uses: chrnorm/deployment-status@releases/v1
        name: Deployment progress
        with:
          token: '${{ github.token }}'
          environment: development
          state: in_progress
          deployment_id: ${{ steps.deployment.outputs.deployment_id }}
```

https://github.com/authmenow/releases/runs/630056476?check_suite_focus=true

I think it is because `in_progress` status requires custom media type: https://developer.github.com/v3/repos/deployments/#parameters-2

**Comment by flemeur** [comment]
I'm also experiencing this issue. I't looks like a fix for it was actually merged in through this PR https://github.com/chrnorm/deployment-status/pull/6 But it was merged into the release branch instead of master. @chrnorm Could you merge it into master and make a new release please?

**Comment by chrnorm** [comment]
@sergeysova @flemeur I've merged the commit from PR #6 into master and created a new release, which I've published to the GitHub Actions marketplace here: https://github.com/marketplace/actions/deployment-status. Thanks for your assistance through raising this issue.

To fix to the specific version you can use `chrnorm/deployment-status@v1.0.0` in your workflows, alternative feel free to continue using the release branch `chrnorm/deployment-status@releases/v1`.
